sky/shell/platform/ios/platform_view_ios.mm

```diff
@@ -0,0 +1,291 @@
+
+  bool UpdateStorageSizeIfNecessary() {
+    GLintSize size([layer_.get() bounds].size);
+
+    if (size == storage_size_) {
+      // Nothing to since the stoage size is already consistent with the layer.
+      return true;
+    }
+
+    if (![EAGLContext setCurrentContext:context_]) {
+      return false;
+    }
+
+    DCHECK(glGetError() == GL_NO_ERROR);
+
+    glBindFramebuffer(GL_FRAMEBUFFER, framebuffer_);
+
+    glBindRenderbuffer(GL_RENDERBUFFER, colorbuffer_);
+    DCHECK(glGetError() == GL_NO_ERROR);
+
+    if (![context_.get() renderbufferStorage:GL_RENDERBUFFER
+                                fromDrawable:layer_.get()]) {
+      return false;
+    }
+
+    GLint width = 0;
+    GLint height = 0;
+    bool rebind_color_buffer = false;
+
+    if (depthbuffer_ != GL_NONE || stencilbuffer_ != GL_NONE ||
+        depth_stencil_packed_buffer_ != GL_NONE) {
+      // Fetch the dimensions of the color buffer whose backing was just updated
+      // so that backing of the attachments can be updated
+      glGetRenderbufferParameteriv(GL_RENDERBUFFER, GL_RENDERBUFFER_WIDTH,
+                                   &width);
+      DCHECK(glGetError() == GL_NO_ERROR);
+
+      glGetRenderbufferParameteriv(GL_RENDERBUFFER, GL_RENDERBUFFER_HEIGHT,
+                                   &height);
+      DCHECK(glGetError() == GL_NO_ERROR);
+
+      rebind_color_buffer = true;
+    }
+
+    if (depth_stencil_packed_buffer_ != GL_NONE) {
+      glBindRenderbuffer(GL_RENDERBUFFER, depth_stencil_packed_buffer_);
+      glRenderbufferStorage(GL_RENDERBUFFER, GL_DEPTH24_STENCIL8_OES, width,
+                            height);
+      DCHECK(glGetError() == GL_NO_ERROR);
+    }
+
+    if (depthbuffer_ != GL_NONE) {
+      glBindRenderbuffer(GL_RENDERBUFFER, depthbuffer_);
+      glRenderbufferStorage(GL_RENDERBUFFER, GL_DEPTH_COMPONENT16, width,
+                            height);
+      DCHECK(glGetError() == GL_NO_ERROR);
+    }
+
+    if (stencilbuffer_ != GL_NONE) {
+      glBindRenderbuffer(GL_RENDERBUFFER, stencilbuffer_);
+      glRenderbufferStorage(GL_RENDERBUFFER, GL_STENCIL_INDEX8, width, height);
+      DCHECK(glGetError() == GL_NO_ERROR);
+    }
+
+    if (rebind_color_buffer) {
+      glBindRenderbuffer(GL_RENDERBUFFER, colorbuffer_);
+      DCHECK(glGetError() == GL_NO_ERROR);
+    }
+
+    storage_size_ = GLintSize(width, height);
+    DCHECK(glCheckFramebufferStatus(GL_FRAMEBUFFER) == GL_FRAMEBUFFER_COMPLETE);
+
+    return true;
+  }
+
+  DISALLOW_COPY_AND_ASSIGN(IOSGLContext);
+};
+
+PlatformView* PlatformView::Create(const Config& config,
+                                   SurfaceConfig surface_config) {
+  return new PlatformViewIOS(config, surface_config);
+}
+
+PlatformViewIOS::PlatformViewIOS(const Config& config,
+                                 SurfaceConfig surface_config)
+    : PlatformView(config, surface_config), weak_factory_(this) {}
+
+PlatformViewIOS::~PlatformViewIOS() {
+  base::mac::ScopedNSAutoreleasePool pool;
```

Comment:
Nope. `= default`ed. Have also added pools around objc calls in `IOSGLContext` since it may be on a thread without an implicit pool.